library/hawkular_alerts_group_trigger.py

```diff
@@ -260,6 +326,7 @@ def main():
     enabled    = module.params['enabled']
     verify_ssl = module.params['verify_ssl']
     ca_file    = module.params['ca_file_path']
+    conditions = module.params['group_conditions']
```

Comment:
I have only one `trigger_mode` but it has to come together with the conditions, otherwise doesn't have any effect.
The user can still create a group trigger without conditions, but then there's no logic in passing the `trigger_mode`